lib/explorer/backend/lazy_series.ex

```diff
@@ -73,6 +73,8 @@ defmodule Explorer.Backend.LazySeries do
     window_sum: 5,
     window_standard_deviation: 5,
     ewm_mean: 5,
+    ewm_std: 6,
+    ewm_var: 6,
```

Comment:
We call these things `variance` and `standard_deviation` below. Should we rename these two to match them?

Reply:
The function names were copied as is from `polars`. We should rename to match the convention used in `Explorer`.